protocols/234495/234495.ot2.apiv2.py

```diff
@@ -0,0 +1,391 @@
+from opentrons import types
+
+metadata = {
+    'protocolName': '''LC-MS Sample Prep: part 1 -
+                       Standards/Calibration Curves''',
+    'author': 'Steve Plonk <protocols@opentrons.com>',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    # get parameter values from json above
+    [labware_tips20, labware_tips300, labware_tuberack, clearance_meoh_water,
+     clearance_dil_dispense, touch_radius, touch_v_offset, track_start,
+     clearance_tfa, clearance_mecn] = get_values(  # noqa: F821
+      'labware_tips20', 'labware_tips300', 'labware_tuberack',
+      'clearance_meoh_water', 'clearance_dil_dispense', 'touch_radius',
+      'touch_v_offset', 'track_start', 'clearance_tfa', 'clearance_mecn')
+
+    ctx.set_rail_lights(True)
+
+    # tips, p20 multi gen2, p300 multi gen2
+    tips20 = [ctx.load_labware(
+     labware_tips20, str(slot)) for slot in [2]]
+    p20s = ctx.load_instrument(
+        "p20_single_gen2", 'left', tip_racks=tips20)
+    tips300 = [ctx.load_labware(labware_tips300, str(slot)) for slot in [6]]
+    p300s = ctx.load_instrument(
+        "p300_single_gen2", 'right', tip_racks=tips300)
+
+    """
+    helper functions
+    """
+
+    def pause_attention(message):
+        ctx.set_rail_lights(False)
+        ctx.delay(seconds=10)
+        ctx.pause(message)
+        ctx.set_rail_lights(True)
+
+    def slow_tip_withdrawal(current_pipette, well_location, to_center=False):
+        if current_pipette.mount == 'right':
+            axis = 'A'
+        else:
+            axis = 'Z'
+        ctx.max_speeds[axis] = 10
+        if to_center is False:
+            current_pipette.move_to(well_location.top())
+        else:
+            current_pipette.move_to(well_location.center())
+        ctx.max_speeds[axis] = None
```

Comment:
Would love to know more about this implementation

Reply:
The purpose is to remove the tip from the liquid at reduced speed to lessen the transporting of liquid away on the exterior surface of the tip (happens to a greater degree with viscous liquids). Reduced speed can end by default at the top of the well or alternatively in the center of the well (useful if the next step occurs in the well center).